### ISSUE TITLE
Firewire / Thunderbolt / Ethernet Support

### ISSUE THREAD

**Issue by Miserlou** [hephaest0s/usbkill#39]
Just got me thinking.. there are plenty of other holes in this machine which actually present a far nastier attack vector than plain USB (DMA!) that currently don't really have any defenses other than superglue.

It'd be super nifty if usbkill 0.2.0 could also support system killing on changes on the Firewire/Thunderbolt/Ethernet ports as well.

**Comment by Miserlou** [comment]
system_profiler SPThunderboltDataType

and

system_profiler SPFireWireDataType

and

system_profiler SPCardReaderDataType

all seem to work just fine, seems like could just apply the same logic and have it work, at least on OSX. udev on linux?

**Comment by pwnsdx** [comment]
Yup. I am aware of these commands, I will add these checks very soon. For Linux, why not udev.

**Comment by nic** [comment]
@hephaest0s Are you think that it's possible?
